workspaces/ui-v2/src/optic-components/hooks/diffs/LearnInitialBodiesMachine.ts

```diff
@@ -114,6 +121,7 @@ export const newInitialBodiesMachine = (
                 assign({
                   allCommands: (ctx) => recomputeCommands(ctx),
                 }),
+                sendUpdate(),
```

Comment:
This is the magic thing from xstate that avoids the hacky thing that i was doing here https://github.com/opticdev/optic/pull/742